example/fluence.yaml

```
@@ -11,8 +11,8 @@ services:
 dependencies:
   npm:
     "@fluencelabs/aqua": 0.7.7-353
-    "@fluencelabs/registry": 0.6.2
     "@fluencelabs/aqua-lib": 0.4.3
+    "@fluencelabs/registry": nightly
```

```suggestion
    "@fluencelabs/registry": latest
```
We don't have for now any automated nightly releases; the current nightly is very outdated